### ISSUE TITLE
Error on https://primacomputing.co.nz/primametro/Scripting.aspx

### ISSUE THREAD

**Issue by MoreporkGithub** [ruffle-rs/ruffle#2383]
# Error Info
Error name: TypeError
Error message: Failed to execute 'compile' on 'WebAssembly': HTTP status code is not ok
Error stack:
```
TypeError: Failed to execute 'compile' on 'WebAssembly': HTTP status code is not ok
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: images/quill1.swf
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined

# Page Info
Page URL: https://primacomputing.co.nz/primametro/Scripting.aspx

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36 Edg/87.0.664.75
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2021-01-11
Channel: nightly
Built: 2021-01-11T00:53:04.486Z
Commit: cce3a8199d48ecbce536a0e49efd501d97a2b370

Sorry, I'm very new to this... if I missed something I should have done please just point me to it.

(dashwood@EOL.co.nz)

Thanks,

MoreporkGitHub.

**Comment by Futurama39** [comment]
From visiting the site, it gives some errors from the server side, #400 might offer some help

**Comment by MoreporkGithub** [comment]
Thanks for your help.

 

I posted a comment in response on Github (I think… 😊 pretty new to all this…)

 

Thanks very much for the link; I followed it and read the thread carefully.

 

I have added a .htaccess file on the same path as the script page but it makes no difference. I’ll try moving that file to the root of the web site and see if it helps.

 

Given that the ruffle engine works on the desktop without problem, I might be able to wrap it in C# and call it from the code-behind for the Script page… But it is a fair bit of work and I am pressed at the moment. I’d much prefer a js solution… 😊

 

Thanks again,

 

Pete.
